test/message/test_runner_describe_it.js

```diff
@@ -339,3 +339,47 @@ describe('timeouts', () => {
     setTimeout(done, 10);
   });
 });
+
+describe('Concurrency option (boolean) = true ', { concurrency: true }, () => {
+  const suiteStartDateTime = new Date().getTime();
+  const extraBufferTime = 10;
```

Comment:
10ms might introduce a lot of flakiness, I am not sure that I would test concurrency with timestamps,
Probably a chain of tests (with a length of `cors * 2` or anything bigger than the number of cors) that wait for each other to complete (each test only completes when next test passes?) might be less flaky

Reply:
also, `test/message` tests the output of the test runner, you probably want to write these tests in `test/parallel